readthedocs/projects/forms.py

```diff
@@ -182,6 +182,16 @@ class Meta(object):
         widget=forms.Textarea,
     )
 
+    def clean_tags(self):
+        tags = self.cleaned_data.get('tags')
+        if tags:
```

Comment:
nitpick: we could just do `.get('tags', [])`, that way we avoid having an extra level of indentation

Reply:
Thanks for the tip.